gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-bin.zip
```

Comment:
Is there any reason to use 6.9 over 6.9.1? The latter's fixes are:

* [#18089] - Deprecate jcenter() repository (Backport to 6.x)
* [#17950] - Renaming and recreating the project directory causes Gradle to lose track of changes on Windows (Backport to 6.x)
* [#17949] - Gradle's up-to-date checks do not work on Windows FAT drives (Backport to 6.x)

Reply:
Agreed, 6.9.1 seems to be a small set of changes that are valuable to some users. While we're updating, it's two more chars, might as well. It is also certainly important that people move away from jcenter(), because it is going to close down entirely soon.

Reply:
Where's my "mark as resolved" button? Anywho, resolved in a5945c2 (dead link but you can still see the commit listed on this page).